README.md

```diff
@@ -107,7 +107,7 @@ Kubernetes questions & development:
 
 ### Adopters
 
-Check out the [list of companies](https://github.com/containers/podman-desktop/blob/adopters/ADOPTERS.md) already using Podman Desktop.
+Check out the [list of companies](https://github.com/containers/podman-desktop/blob/main/ADOPTERS.md) already using Podman Desktop.
```

Comment:
Can't we use relative links (see line 85)

Reply:
fixed